app/packs/stylesheets/decidim/decidim_application.scss

```diff
@@ -23,6 +23,8 @@
 @import "stylesheets/general/tabs.scss";
 @import "stylesheets/general/timeline.scss";
 
+@import "stylesheets/decidims_finder.scss";
```

Comment:
regarding the PR from froger I'm not sure it is a good practice to add the extension when importing CSSs

Reply:
Cool